File: code/old/ancestryscraping.py
## WEB SCRAPING FOR ANCESTRY LIBRARY PORT DATA
import re
from urllib.request import Request, urlopen
import requests
headers={'User-Agent': 'Mozilla/6.0'}
response = requests.get('https://www.ancestrylibrary.com/search/collections/1263/', headers=headers)
response2 = requests.get('https://www.ancestrylibrary.com/search/collections/1263/', headers=headers)

print(response2.text)

File: code/old/ancestryscraping.R
########################################################################
### FILE DESCRIPTION: Scraping Port of Entry Data from Ancestry.com
### CREATED BY: Amy Kim
### CREATED ON: Oct 27 2022
### LAST MODIFIED: Oct 27 2022
########################################################################
library(tidyverse)
library(glue)
library(rvest)
library(RSelenium)

########################################################################
### DEFINING PATHS
########################################################################
if (Sys.getenv("USER") == "amykim"){
  dbox = "/Users/amykim/Dropbox (Princeton)/head_tax_data"
  git = "/Users/amykim/Documents/GitHub/headtax"
}

########################################################################
### WEB SCRAPING
########################################################################
rD <- rsDriver(browser = "chrome", chromever = "88.0.4324.96", port = 10L)
remDr <- rD[["client"]]
remDr$navigate()
















